pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -711,10 +711,11 @@ func nextFlushWithJitter(s timeutil.TimeSource, d time.Duration, j float64) (tim
 	if j < 0 || d < 0 {
 		return s.Now(), errors.AssertionFailedf("invalid jitter value: %f, duration: %s", j, d)
 	}
-	if j == 0 || d == 0 {
+	maxJitter := int64(j * float64(d))
+	if j == 0 || d == 0 || maxJitter == 0 {
```

Comment:
as discussed in https://github.com/cockroachdb/cockroach/pull/144304#discussion_r2042256161:
```suggestion
	if maxJitter == 0 {
```